docs/README.md

```diff
@@ -32,8 +32,8 @@ to the `gh-pages` branch and push it to GitHub.
 
 ### Auto-deploying
 
-Any new commit on `master` that modifies the `./docs` folder will
-automatically trigger a build and deploy it.
+Any new commit on `master` that modifies the `./docs` folder will automatically
```

Comment:
I'm not sure why those lines have their wrapping changed. Maybe the linter corrected them. I have a PR pending to add an automatic check of the linting, I'm not 100% sure it's working reliably, I'll check.

In the meantime, let's consider this correct; if it isn't, the linter will catch it later.

Reply:
My linter correct them.

Do you want me to amend this part?